pglogical_proto_native.c

```diff
@@ -487,6 +494,7 @@ pglogical_write_tuple(StringInfo out, PGLogicalOutputData *data,
 
 		ReleaseSysCache(typtup);
 	}
+	FreeTupleDesc(tupdesc);
```

Comment:
I was concerned that freeing the template tupledesc here should result in double-frees , because the assignment `tupdesc->attrs[nliveatts] = att;` earlier aliased the `Form_pg_attribute` from the `desc` returned by `RelationGetDescr`. But despite the comment on `FreeTupleDesc` that it frees the TupleDesc including "all substructure" it doesn't appear to try to free the `attrs` member values.